tests/frontend/org/voltdb/TestAdhocCreateTable.java

```
@@ -76,6 +76,23 @@ public void testBasicCreateTable() throws Exception
                 threw = true;
             }
             assertTrue("Shouldn't have been able to create table FOO twice.", threw);
+
+            // make sure cannot create table with default value equals to minimum value
+            String[] testType = {"tinyint", "smallint", "integer", "bigint"};
+            String[] minVal = {"-128", "-32768", "-2147483648", "-9223372036854775808"};
+
+            for (int i = 0; i < testType.length; i++) {
+                threw = false;
+                try {
+                    m_client.callProcedure("@AdHoc",
+                            "create table t (c " + testType[i] + " default " + minVal[i] + ");");
+                }
+                catch (ProcCallException pce) {
+                    assertTrue(pce.getMessage().contains("data exception: numeric value out of range"));
+                    threw = true;
+                }
+                assertTrue("Create table with default minimum value will throw exception.", threw);
```

Creating numeric table column using the reserved minimum value as default should throw an exception.